README.md

```diff
@@ -55,7 +55,16 @@ for details on submitting patches and the contribution workflow.
 
 ## Follow us
 
-Twitter: [@PingCAP](https://twitter.com/PingCAP)
+### Twitter 
+[@PingCAP](https://twitter.com/PingCAP)
+
+### Mailing lists
+
+Users of TiDB trade notes on:
```

Comment:
I don't think we need this sentence... After all, we only have one Google Group...